columbia-html.html

```diff
@@ -419,24 +458,24 @@ <h3 id="tables">8. Tables</h3>
   <blockquote>
     <table border="">
       <tr>
-        <th>Heading A</th>
-        <th>Heading B</th>
-        <th>Heading C</th>
+        <th style="background-color: #FFC0CB" class="table-red">Heading A</th>
```

Comment:
itt ezeket ossze lehetne vonni ugy, hogy a `background-color`-t bele tudod irni a `table-red` classbe es nem kell utana folyamatosan mindig leirni

Reply:
Azért lett így készítve, hogy beletegyek minden féle megoldást a kódba, hogy lássátok, hogy több féle eljárást is elsajátítottam.